openpype/hosts/houdini/plugins/create/create_pointcache.py

```diff
@@ -37,14 +36,30 @@ def create(self, subset_name, instance_data, pre_create_data):
         }
 
         if self.selected_nodes:
-            parms["sop_path"] = self.selected_nodes[0].path()
+            parent = self.selected_nodes[0]
+            parms["sop_path"] = parent.path()
+
+            child_render = ""
+            child_output = ""
 
             # try to find output node
-            for child in self.selected_nodes[0].children():
+            for child in parent.children():
```

Comment:
if user wants to select nodes(aka children nodes) inside geometry node and create instance, it will break at some point as it only allows users to select Geometry Nodes.